### PR TITLE
Add document for tiup cluster clean and rename

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)
Add document for tiup cluster clean and rename.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4233; https://github.com/pingcap/docs-cn/pull/4273
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by TomShawn** [comment]
@lilin90 Please help review this PR, thanks!

**Review by lucklove — APPROVED** [review]
LGTM

File: maintain-tidb-using-tiup.md [hunk 1]
```diff
@@ -211,6 +226,56 @@ Similar to the `start` command, the `stop` command supports stopping some of the
...
+
+- Clean up the logs and data of all services except Prometheus:
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    tiup cluster clean ${cluster-name} --all --ignore-role prometheus
+    ```
+
+- Clean up the logs and data of all services except the node `172.16.13.11:9000`:
```
Comment (lilin90) [inline maintain-tidb-using-tiup.md:263]:
```suggestion
- Clean up the logs and data of all services except the `172.16.13.11:9000` instance:
```

File: maintain-tidb-using-tiup.md [hunk 2]
```diff
@@ -211,6 +226,56 @@ Similar to the `start` command, the `stop` command supports stopping some of the
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    tiup cluster clean ${cluster-name} --all --ignore-node 172.16.13.11:9000
+    ```
+
+- Clean up the logs and data of all services except the node `172.16.13.12`:
```
Comment (lilin90) [inline maintain-tidb-using-tiup.md:271]:
```suggestion
- Clean up the logs and data of all services except the `172.16.13.12` node:
```

**Review by lilin90 — CHANGES_REQUESTED** [review]

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**